frontend/src/components/common/SimpleTable.tsx

```diff
@@ -155,6 +272,15 @@ export default function SimpleTable(props: SimpleTableProps) {
                     {...otherProps}
                   >
                     {label}
+                    {sort &&
+                    <ColumnSortButtons
+                      defaultSortingColumn={defaultSortingColumn}
+                      isIncreasingOrder={isIncreasingOrder}
+                      setIsIncreasingOrder={setIsIncreasingOrder}
+                      sortColIndex={i}
+                      setSortColIndex={setSortColIndex}
```

Comment:
Maybe we just have to pass it `isIncreasingOrder` (for the right icon to be shown) and then use a `onClick` and just have the callback use the column for sorting here (instead of passing 5 properties).